react/src/pages/EndpointDetailPage.tsx

```diff
@@ -522,6 +609,227 @@ const EndpointDetailPage: React.FC<EndpointDetailPageProps> = () => {
+                        endpoint?.status,
+                      ) ||
+                      (!!endpoint?.created_user_email &&
+                        endpoint?.created_user_email !== currentUser.email)
+                        ? {
+                            color: token.colorTextDisabled,
+                          }
+                        : {
+                            color: token.colorInfo,
+                          }
+                    }
+                    disabled={
+                      isDestroyingStatus(
+                        endpoint?.replicas ?? endpoint?.desired_session_count,
+                        endpoint?.status,
+                      ) ||
+                      (!!endpoint?.created_user_email &&
+                        endpoint?.created_user_email !== currentUser.email)
+                    }
+                    onClick={() => {
+                      if (row) {
+                        setEditingAutoScalingRule(row);
+                        setIsOpenAutoScalingRuleModal(true);
+                      }
+                    }}
+                  />
+                  <Popconfirm
+                    title={t('dialog.warning.CannotBeUndone')}
+                    okText={t('button.Delete')}
+                    okButtonProps={{
+                      danger: true,
+                    }}
+                    disabled={isInFlightDeleteAutoScalingRuleMutation}
+                    onConfirm={() => {
+                      if (autoScalingRules) {
+                        commitDeleteAutoScalingRuleMutation({
+                          variables: {
+                            id: row?.id as string,
+                          },
+                          onCompleted: (res, errors) => {
+                            if (
+                              !res?.delete_endpoint_auto_scaling_rule_node?.ok
+                            ) {
+                              message.error(
+                                res?.delete_endpoint_auto_scaling_rule_node
+                                  ?.msg,
+                              );
+                            } else if (errors && errors.length > 0) {
+                              const errorMsgList = _.map(
+                                errors,
+                                (error) =>
+                                  error.message || t('dialog.ErrorOccurred'),
+                              );
+                              for (const error of errorMsgList) {
+                                message.error(error, 2.5);
+                              }
+                            } else {
+                              setEditingAutoScalingRule(null);
+                              startRefetchTransition(() => {
+                                updateFetchKey();
+                              });
+                              message.info({
+                                key: 'autoscaling-rule-deleted',
```

Comment:
For success message, please use `message.success` instead of `message.info`
```suggestion
                              message.success({
```